src/js/polyfills/url.js

```diff
@@ -4,3 +4,31 @@ import { URL, URLSearchParams } from 'whatwg-url';
 globalThis.URL = URL;
 globalThis.URLPattern = URLPattern;
 globalThis.URLSearchParams = URLSearchParams;
+
+let _objectURLs;
+Object.defineProperty(globalThis, 'objectURLs', {
+    enumerable: false,
+    configurable: false,
+    get: () => {
+        if (!_objectURLs) {
+            _objectURLs = new Map();
+        }
+        return _objectURLs;
+    }
+});
+
+// TODO: straggly, without the wrapped console.log, the runtime will abort
```

Comment:
The problem is different now, the function is spammed. I made a small video capture to show you:
[objecturl_bug.webm](https://github.com/saghul/txiki.js/assets/37633694/c1911300-1bd9-4657-9072-0870b301cc1b)
Therefore, with the current case, I have no backtrace to provide. And I have no clue what is that printed SymtaxError, since there is no problem at all when running it with the empty `console.log('')`.